projectile.el

```diff
@@ -83,7 +83,7 @@
 (declare-function ggtags-update-tags "ext:ggtags")
 (declare-function ripgrep-regexp "ext:ripgrep")
 (declare-function rg-run "ext:rg")
-(declare-function vterm "ext:vterm")
+(declare-function vterm-other-window "ext:vterm-other-window")
```

Comment:
You should not change the ext part - that's the library name (`vterm.el`).

Reply:
Sorry about that. I haven't written emacs lisp more than a few simple functions for my config.

Thanks a lot for the feedback. Fixed now.